homeassistant/components/websocket_api/__init__.py

```diff
@@ -16,6 +16,7 @@
 from .const import (  # noqa: F401
     ERR_HOME_ASSISTANT_ERROR,
     ERR_INVALID_FORMAT,
+    ERR_NOT_ALLOWED,
```

Comment:
`ERR_NOT_ALLOWED` is referenced by `assist_pipeline`, `homeassistant` and `thread` components